### PR TITLE
Added more weather commentary based on temperature Issue #14

### PR DESCRIPTION
**Pull request by rushilShetty**
Added more commentary based on how hot or cold the temperature is going to be

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -78,7 +88,12 @@ def weather(city, state):
     # + degree_sign + " and there is a " + RAIN_WARNING)
     location = {'city': city, 'state': state}
 
-    weather_info = {'temperature': temperature, 'rain': rain_commentary}
+    temperature = str(temperature)
+    if not rain_commentary:
+        weather_info = {'temperature': temperature, 'commentary': rain_commentary}
+    else:
+        weather_info = {'temperature': temperature, 'commentary': commentary}
```
Comment (M0nica) [inline app.py]:
```suggestion
        weather_info = {'temperature': temperature, 'commentary': rain_commentary}
```

File: app.py [hunk 2]
```diff
@@ -78,7 +88,12 @@ def weather(city, state):
     # + degree_sign + " and there is a " + RAIN_WARNING)
     location = {'city': city, 'state': state}
 
-    weather_info = {'temperature': temperature, 'rain': rain_commentary}
+    temperature = str(temperature)
+    if not rain_commentary:
+        weather_info = {'temperature': temperature, 'commentary': rain_commentary}
```
[the remark this answers is not in the document]
Comment (M0nica) [reply app.py]: Should this be commentary and not rain_commentary?

**Review by M0nica — CHANGES_REQUESTED** [review]
Thank you for this PR. I had a question but other than that it looks good to me.

**Comment by rushilShetty** [comment]
Yes it should indeed be commentary and not rain commentary. Good find :)

**Comment by M0nica** [comment]
@rushilShetty can you please update the commentary / rain commentary references? I do not have permission to edit your pull request? Once that is done we can merge 🎉

**Pull request closed without merging**